assignment3/functions.dart

```diff
@@ -0,0 +1,132 @@
+import 'dart:io';
+import 'graph.dart';
+import 'util.dart';
+
+void printOptions() {
+  print('Enter the as per options below');
+  print('1 : Add new node to the tree');
+  print('2 : Add dependency');
+  print('3 : Get the immediate parents of a node');
+  print('4 : Get the immediate children of a node');
+  print('5 : Get all the ancestors of a node');
+  print('6 : Get all the descendants of a node');
+  print('7 : Delete a node from tree');
+  print('8 : exit');
+}
+
+int takeUserChoice() {
+  int validChoice = Util.validUserInput();
+
+  while (validateUserChoice(validChoice) == false) {
+    printOptions();
+    validChoice = Util.validUserInput();
+  }
+  return validChoice;
+}
+
+int validGraphId() {
```

Comment:
Add proper name according to its function

Reply:
changed it to validateGraphId